extension/src/background/messageListener/popupMessageListener.ts

```diff
@@ -429,30 +429,32 @@ export const popupMessageListener = (request: Request) => {
       // we have cleared redux store via reloading extension/browser
       // construct allAccounts from local storage
       // log the user in using all accounts and public key/phrase from above to create the store
-      await Promise.all(
-        keyIdList.map(async (keyId: string) => {
-          let keyStore;
-
-          // iterate over each keyId we have and get the associated keystore
-          try {
-            keyStore = await keyManager.loadKey(keyId, password);
-          } catch (e) {
-            console.error(e);
-          }
-
-          if (keyStore) {
-            // push the data into a list of accounts
-
-            const { publicKey, extra = { mnemonicPhrase: "" } } = keyStore;
-            const { imported = false } = extra;
-            unlockedAccounts.push({
-              publicKey,
-              name: accountNameList[keyId] || `Account ${keyIdList.length}`,
-              imported,
-            });
-          }
-        }),
-      );
+
+      // eslint-disable-next-line no-plusplus
```

Comment:
all that async iteration talk got me thinking maybe I should ensure order here afterall just to account for any weird edge cases :)

Reply:
Sweet! 🙂 Simple `for` loop for the win! 👍